### PR TITLE
[FIX, PARAM] fixed comets and OMSSAs default tolerance.

### PR DESCRIPTION
**Pull request by timosachsenberg**
unified several precursor mass tolerance parameters of tools and fixed those that are pretty off

### CODE REVIEW

**Comment by jpfeuffer** [comment]
I like the change! It will break some workflows though.

File: src/topp/OMSSAAdapter.cpp [hunk 1]
```diff
@@ -201,8 +201,9 @@ class TOPPOMSSAAdapter :
     registerOutputFile_("out", "<file>", "", "Output file ");
     setValidFormats_("out", ListUtils::create<String>("idXML"));
 
-    registerDoubleOption_("precursor_mass_tolerance", "<tolerance>", 1.5, "Precursor mass tolerance (Default: Dalton)", false);
-    registerFlag_("precursor_mass_tolerance_unit_ppm", "If this flag is set, ppm is used as precursor mass tolerance unit");
+    registerDoubleOption_("precursor_mass_tolerance", "<tolerance>", 10.0, "Precursor mass tolerance (Default: Dalton)", false);
```
Comment (jpfeuffer) [inline src/topp/OMSSAAdapter.cpp]: mono mass?

File: src/topp/MyriMatchAdapter.cpp [hunk 2]
```diff
@@ -223,9 +223,9 @@ class MyriMatchAdapter :
     setValidFormats_("in", ListUtils::create<String>("mzML"));
     registerOutputFile_("out", "<file>", "", "Output file");
     setValidFormats_("out", ListUtils::create<String>("idXML"));
-    registerDoubleOption_("precursor_mass_tolerance", "<tolerance>", 1.5, "Precursor mono mass tolerance.", false);
+    registerDoubleOption_("precursor_mass_tolerance", "<tolerance>", 10.0, "Precursor mono mass tolerance.", false);
```
Comment (jpfeuffer) [inline src/topp/MyriMatchAdapter.cpp]: Unify expressions in all the parameters?

File: src/topp/OMSSAAdapter.cpp (hunk 1 above)
Comment (timosachsenberg) [inline src/topp/OMSSAAdapter.cpp]: ups default is now ppm. what do you mean with mono mass?

Comment (jpfeuffer) [inline src/topp/OMSSAAdapter.cpp]: In all the other descriptions you specifically mention that it is the monoisotopic mass that is specified here.

Comment (timosachsenberg) [inline src/topp/OMSSAAdapter.cpp]: ah I just saw it

Comment (timosachsenberg) [inline src/topp/OMSSAAdapter.cpp]: Ok can do

**Pull request merged**